src/components/label/label.tsx

```diff
@@ -58,18 +59,23 @@ export class Label {
   /**
    * @internal
    */
-  @Event({ bubbles: false }) calciteInternalLabelClick: EventEmitter<{
+  @Event({ bubbles: false }) labelClickEvent: EventEmitter<{
     sourceEvent: MouseEvent;
   }>;
 
+  /**
+   * @internal
+   */
+  @Event() labelConnectedEvent: EventEmitter;
```

Comment:
The label is dispatching the events directly on the document, so this can be removed.